lib/parser.js

```diff
@@ -98,9 +100,18 @@ function _mapSymbol(symbol){
     symbol.type = types[0].types.join(' | ');
   }
 
-  if(returns.length === 1){
-    symbol["return"] = returns[0].types.join(' ');
-    symbol["returnDescription"] = returns[0].description.replace(/(<([^>]+)>)/ig, '');
```

Comment:
Because the possibility exists for multiple returns. For instance, see [this file](https://github.com/pattern-library/pattern-importer/blob/master/lib/import-single-pattern.js) where the returned object is broken down into its various parts. With a single return/returnDescription there is no way to do that, each subsequent return would overwrite the last. Bad coding from me in the first place.

Also, for my second go at this file, I wanted to clean up my changes, making them more uniform - thus the change from isReturn/isRequires to hasReturn/hasRequires as well. And now the return section matches the way examples & requires work.